### ISSUE TITLE
Update documentation: website is moving to ghostery.com/whotracksme

### ISSUE THREAD

**Issue by philipp-classen** [whotracksme/whotracks.me#367]
The website has been integrated into https://www.ghostery.com/whotracksme now.

Data dumps are not affected, but the links to the old whotracks.me website should be updated to use the new links:
* https://www.ghostery.com/whotracksme
* https://www.ghostery.com/whotracksme/trackers
* https://www.ghostery.com/whotracksme/websites
* https://www.ghostery.com/whotracksme/explorer

Also, the README could do a better job at explaining the interaction between the related projects:
* https://github.com/whotracksme/whotracks.me (this repository)  
* https://github.com/ghostery/trackerdb

In a nutshell, TrackerDB provides the meta-data (e.g. company descriptions), while the whotracks.me repository is for the actual data (both in its raw form as CSV files, but also in visualization form as shown on the website).

Note: The website migration has no implication on other aspects like licensing or the way we publish monthly data dumps. Building the website with locally will be still possible, even if the transition to the new website is completed.

**Comment by philipp-classen** [comment]
Changed now: whotracks.me redirects now to https://www.ghostery.com/whotracksme/
